luigi/contrib/bigquery_avro.py

```diff
@@ -0,0 +1,78 @@
+"""Specialized tasks for handling Avro data in BigQuery from GCS.
+"""
+import avro
```

Comment:
I guess this would have to go in a `try` block following our current convention? (Have there been talks about longer term breaking the contrib modules into separate packages with dependencies properly declared in their `setup.py`s?)